### PR TITLE
corrected wrong offset calculation

### PR DESCRIPTION
**Pull request by rozek**
while experimenting with "draggable", I noticed a wrong calculation of "offset.parent" - here is my fix for that bug

### CODE REVIEW

File: src/sortable/sortable.js [hunk 1]
```diff
@@ -378,9 +378,9 @@ export default class Sortable extends Draggable {
 
 		if (
 			(revert === 'invalid' && !sortStop.droppable) ||
-			(revert === 'valid' && sortStop.droppable) ||
-			revert ||
-			(isFunction(revert) && revert(this.currentItem, sortStop.droppable))
+			(revert === 'valid'   && sortStop.droppable)  ||
+			(isFunction(revert)   && revert(this.currentItem, sortStop.droppable)) ||
+			(revert == true)
```
Comment (marcospont) [inline src/sortable/sortable.js:383]: Fix will be incorporated

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "agnostic-draggable",
-	"version": "1.0.5",
+	"version": "1.0.6",
```
Comment (marcospont) [inline package.json:3]: Version bump needs to be done using standard-version npm package

File: .gitignore [hunk 3]
```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+
+.DS_Store
```
Comment (marcospont) [inline .gitignore:106]: Change will be incorporated.

File: src/draggable/draggable.js [hunk 4]
```diff
@@ -414,9 +414,9 @@ export default class Draggable {
 
 		if (
 			(revert === 'invalid' && !dragStop.droppable) ||
-			(revert === 'valid' && dragStop.droppable) ||
-			revert ||
-			(isFunction(revert) && revert(this.element, dragStop.droppable))
+			(revert === 'valid'   && dragStop.droppable)  ||
+			(isFunction(revert)   && revert(this.element, dragStop.droppable)) ||
```
Comment (marcospont) [inline src/draggable/draggable.js:418]: Fix will be incorporated

File: src/sortable/sortable.js [hunk 5]
```diff
@@ -627,9 +627,9 @@ export default class Sortable extends Draggable {
 			} else if (nodeName === 'img') {
 				element.setAttribute('src', this.currentItem.getAttribute('src'));
 			}
-			style(element, {
-				visibility: 'hidden'
-			});
+			//style(element, {
+			//	visibility: 'hidden'
```
Comment (marcospont) [inline src/sortable/sortable.js:631]: Can you elaborate about this one?

Comment (rozek) [reply src/sortable/sortable.js:631]: FYI: today, I did not find time to answer your question - I'll do so by tomorrow. Sorry for the delay...

Comment (rozek) [reply src/sortable/sortable.js:631]: Oh, I just looked into the code and remembered: I had to make the placeholder visible in order to see where the dragged list element could/should be inserted. Unless you have a better alternative, I'd recommend to keep the placeholder visible...

**Pull request merged**